### ISSUE TITLE
GitHub code mapping does not work in Next.js server side code

### ISSUE THREAD

**Issue by alvarlagerlof** [getsentry/sentry#47444]
### Environment

SaaS (https://sentry.io/)

### Link

https://alvar-lagerlof.sentry.io/issues/4095244054/?project=4505020218277888&referrer=issue-stream&stream_index=1

### DSN

https://85e3ebd203224c8ba6c6a2832b5ab1f9@o4505020211134464.ingest.sentry.io/4505020218277888

### Steps to Reproduce

1. Deploy a Next.js project
2. Cause a server side error
3. Open the issue
4. There is no way to see what line it is coming from in GitHub like you can for client side code

### Expected Result

There is a "Open this line in GitHub" link.

### Actual Result

Client:

<img width="865" alt="image" src="https://user-images.githubusercontent.com/14835120/232313543-7c00eb60-8298-487f-8f28-f054625676c7.png">

Server:

<img width="856" alt="image" src="https://user-images.githubusercontent.com/14835120/232313555-50bbb6c7-2096-4759-a821-7e73e8db69ad.png">

**Comment by lforst** [comment]
@getsentry/issue-experience What is necessary to make this work?

**Comment by scefali** [comment]
@lforst For some reason, the value of `inApp` is false for that stack frame which is why we don't show the open-in line. I think this is an issue on the SDK side.

**Comment by lforst** [comment]
@scefali yeah i know thats why i am asking what we need to do to get this working

**Comment by snigdhas** [comment]
@getsentry/team-web-sdk-frontend Is there something on the sdk side or some configuration to get the frame to parse as `inApp`?

**Comment by malwilley** [comment]
@lforst AFAICT if the SDK sends `in_app: true` for the stack frame this should work correctly. Let us know if that's not enough information or if we need to do something on the product side.

**Comment by lforst** [comment]
I think this is an issue in our stack trace parsing 🤔 https://github.com/getsentry/sentry-javascript/pull/8008
